dependency-versions/pom.xml

```diff
@@ -130,7 +130,7 @@
         <mojo-executor.vespa.version>2.4.0</mojo-executor.vespa.version>
         <netty.vespa.version>4.1.112.Final</netty.vespa.version>
         <netty-tcnative.vespa.version>2.0.65.Final</netty-tcnative.vespa.version>
-        <onnxruntime.vespa.version>1.19.2</onnxruntime.vespa.version>
+        <onnxruntime.vespa.version>1.20.0</onnxruntime.vespa.version>
```

Comment:
good catch but this time it was intentional, because the maven artifacts for 1.20.1 don't exist - there were no changes to the java part between 1.20.0 and 1.20.1, so I'm guessing that's the reason.